### PR TITLE
Polish diamond operator usage

### PR DESCRIPTION
**Pull request by ngocnhan-tran1996**

### CODE REVIEW

File: web/src/main/java/org/springframework/security/web/authentication/LoginUrlAuthenticationEntryPoint.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004, 2005, 2006 Acegi Technology Pty Limited
```
Comment (jzheaux) [inline web/src/main/java/org/springframework/security/web/authentication/LoginUrlAuthenticationEntryPoint.java:2]: It appears this update is not about improving diamond operator usage. If you want to keep it in, please do two things:

1. Leave any Acegi copyrights as-is
2. Move this to a separate commit for easier backporting

**Review by jzheaux — APPROVED** [review]

File: web/src/main/java/org/springframework/security/web/authentication/LoginUrlAuthenticationEntryPoint.java (hunk 1 above)
Comment (ngocnhan-tran1996) [reply web/src/main/java/org/springframework/security/web/authentication/LoginUrlAuthenticationEntryPoint.java]: I think I keep Acegi copyrights and just remove redundant keyword

**Comment by jzheaux** [comment]
Thanks! This is now merged into `main`.
